### ISSUE TITLE
 VM list command failed with exit status 1 - VM Manager disabled

### ISSUE THREAD

**Issue by 13robin37** [domalab/ha-unraid#18]
**Describe the bug**
Error logs in HA show "VM list command failed with exit status 1" although I disabled the VM manager in Unraid.

**Expected behavior**
When Enable VMs = No in Unraid, there shouldn't be a check if any VMs are running.

**Screenshots**
![SCR-20241120-iwoz](https://github.com/user-attachments/assets/1c3ea16b-fcec-425a-bfb1-2c0b8cb0f1ef)

**Unraid (please complete the following information):**
 - Version 6.12.13

**Additional context**
-

**Comment by domalab** [comment]
@13robin37 Thanks for reporting the bug.

**Comment by domalab** [comment]
@13robin37 I have updated the code to check the vm service status. If it's stopped then it won't create vm switch entities. Feel free to test it on your system and let me know if it works for you.

I haven't officially released a new version yet as I'm still testing other fixes I implemented.

Copy the code files from unraid folder in repo to your custom_components\unraid folder and restart home assistant.

Best to test it on HA Dev instance if you have one
